SurveyApp/Services/Network/API/TwoDecodableResponseSerializer.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TwoDecodableResponseSerializer.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/30/21.
+//
+
+import Foundation
+import Alamofire
+
+final class TwoDecodableResponseSerializer<T: Decodable>: ResponseSerializer {
```

Comment:
Why is this called `TwoDecodableResponseSerializer`?

Reply:
Because it uses Alamofire's `DecodableResponseSerializer` to decode both success and error cases.
What name do you think would be suitable for this class? please help suggest 🙇‍♀️

Reply:
makes sense. I think `APIResponseSerializer` or `SurveyAPIResponseSerializer` might be clearer, not sure. 🤔

Reply:
`APIResponseSerializer` would be clearer. I'll make that change. Thank you 🙏 😄